charts/gateway-helm/templates/certgen-rbac.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
   {{- end }}
   annotations:
     "helm.sh/hook": pre-install
```

Comment:
piggying another request in this PR, can we add `pre-upgrade` to the above line for consistency